### PR TITLE
Add badges for Slack and docs

### PR DESCRIPTION
**Pull request by bhavin192**
- This makes is really easy to find the website as well as Slack when
  someone visits the repository

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # BotKube
-[![Build Status](https://travis-ci.org/infracloudio/botkube.svg?branch=master)](https://travis-ci.org/infracloudio/botkube) [![Go Report Card](https://goreportcard.com/badge/github.com/infracloudio/botkube)](https://goreportcard.com/report/github.com/infracloudio/botkube)
+[![Build Status](https://travis-ci.org/infracloudio/botkube.svg?branch=master)](https://travis-ci.org/infracloudio/botkube) [![Go Report Card](https://goreportcard.com/badge/github.com/infracloudio/botkube)](https://goreportcard.com/report/github.com/infracloudio/botkube) [![BotKube website](https://img.shields.io/badge/docs-botkube.io-blue.svg)](https://botkube.io) [![Slack](https://botkube-slack.herokuapp.com/badge.svg)](http://join.botkube.io/)
```
Comment (PrasadG193) [inline README.md]: @bhavin192 Can you change badge url from `https://botkube.io` -> `https://www.botkube.io`? https://botkube.io is not reachable at the moment.

Comment (bhavin192) [reply README.md]: I think we should make the domain reachable instead. I think both botkube.io and www.botkube.io should point to the website.

Comment (bhavin192) [reply README.md]: The PR is already merged, if you think that's the correct approach please do it directly to `develop` branch :smile:

Comment (PrasadG193) [reply README.md]: Yeah, let's fix this in domain config :)

**Pull request merged**